compiler/graph_builder.h

```diff
@@ -43,13 +44,15 @@ class GraphBuilder {
     Node* MOp(const onnx::NodeProto& base, const std::vector<Value*>& inputs, const std::vector<Value*>& outputs);
 
     Value* Const(const chainerx::Array& ary, Value* value = nullptr);
-
     template <class T>
-    Value* Const(const Type& type, const std::vector<T>& data, Value* value = nullptr);
-
+    Value* ScalarConst(const T& v, Value* value = nullptr) {
```

Comment:
As chatted locally, I'd prefer removing this since it's easy to misuse this function